packages/module/src/AttachmentEdit/AttachmentEdit.scss

```diff
@@ -0,0 +1,14 @@
+.pf-chatbot__attachment-language {
+  color: var(--pf-t--color--gray--50);
```

Comment:
ideally, we want to avoid using tokens with a number at the end in CSS overrides (like `--50`).

Instead we should use semantic tokens which most closely map to our usage of the token. So in this case, you'll want to find a text color token like `--pf-t--global--text--color--regular` or another one which gets you closest to the text color you're looking for.

this URL is likely the most helpful for finding appropriate tokens.
https://staging-v6.patternfly.org/tokens/all-patternfly-tokens

LMK if you want help tracking down appropriate tokens.

Reply:
I had a go, but let me know if you want a different token here! It's a lighter gray than the regular text I think.